src/ctap/response.rs

```diff
@@ -301,13 +305,22 @@ mod test {
             algorithms: None,
             default_cred_protect: None,
             #[cfg(feature = "with_ctap2_1")]
+            min_pin_length: 4,
+            #[cfg(feature = "with_ctap2_1")]
             firmware_version: None,
         };
         let response_cbor: Option<cbor::Value> =
             ResponseData::AuthenticatorGetInfo(get_info_response).into();
+        #[cfg(not(feature = "with_ctap2_1"))]
+        let expected_cbor = cbor_map_options! {
+            0x01 => cbor_array_vec![vec!["FIDO_2_0"]],
+            0x03 => vec![0x00; 16],
+        };
+        #[cfg(feature = "with_ctap2_1")]
         let expected_cbor = cbor_map_options! {
             0x01 => cbor_array_vec![vec!["FIDO_2_0"]],
```

Comment:
Maybe using a variable for the versions, so that it's clear it's the same value.

```rust
let versions = vec!["FIDO_2_0".to_string()];
let get_info_response = AuthenticatorGetInfoResponse {
    versions: versions.clone(),
    extensions: None,
    ...
};
let response_cbor = ...;
let expected_cbor = cbor_map_options! {
    0x01 => cbor_array_vec![versions],
    0x03 => vec![0x00; 16],
    #[cfg(feature = "with_ctap2_1")]
    0x0D => 4
};
```

Ping @gendx to also add feature support for the `cbor_map_options` macro.

Reply:
Variable is added, issue is created.